### ISSUE TITLE
Support RDFJS types

### ISSUE THREAD

**Issue by rubensworks** [joachimvh/SPARQLAlgebra.js#3]
I've noticed this nice TODO here: 
```
// TODO: update to rdf js instead of own object
patterns: TriplePattern[];
```

It would be great if a pattern would be an `RDF.Quad` instead, following the RDFJS types from [`@types/rdf-js`](https://www.npmjs.com/package/@types/rdf-js).

Would this be possible in the current codebase?

**Comment by joachimvh** [comment]
Possibly, would be much easier though if SPARQL.js already did that since I just copy the output from there. ;) Otherwise I will have to add some parsing for this.

This request also made me realize that the implementation of quads is not yet completely correct since BGPs only link to TriplePatterns, will have to fix that.

**Comment by joachimvh** [comment]
Fixed in fa8e5a9
